### ISSUE TITLE
delete tasks

### ISSUE THREAD

**Issue by kfox1111** [ga4gh/task-execution-schemas#126]
Is there a standard way to delete tasks when they are no longer of use?

**Comment by kfox1111** [comment]
Is cancel the right api for deletion even in successful ones?

**Comment by adamstruck** [comment]
No, the TES API does not define a standard way to delete tasks that are no longer of use.

**Comment by vsmalladi** [comment]
Might be an optional extension. Lets have further discussion on the need.
